src/UserStories/UserStories.css

```diff
@@ -125,8 +135,24 @@ html {
 .story-actions {
   justify-self: flex-end;
 }
-.story-actions button {
+.story-action {
+  padding: 0;
+  background: none;
+  border: none;
   color: white;
+  cursor: pointer;
+}
+.story-action + .story-action {
+  margin-left: 5px;
+}
+
+.story-body {
+  transition: transform 0.5s ease-out;
+  border-radius: 8px;
+  overflow: hidden;
```

Comment:
Does the story body still need the overflow hidden since it has its own border-radius.

Reply:
I'll check.

Reply:
That's correct, but usually, you have to explain why to the reviewer because they mistakenly think that the overflow is not needed, "yes, it does" won't change their mind.

Reply:
Oh, sure.
We need the overflow here for the border radius.